### PR TITLE
add arm64 support

### PR DESCRIPTION
**Pull request by cgroschupp**

### CODE REVIEW

File: bin/install [hunk 1]
```diff
@@ -27,14 +50,7 @@ install_tfupdate() {
...
 
-    local platform=""
-    platform="$(uname -s)"
-
-    if [ "Darwin" = "$platform" ]; then
-      curl -L -C - "https://github.com/minamijoyo/tfupdate/releases/download/v${version}/tfupdate_${version}_darwin_amd64.tar.gz" | tar zxf - -C "${install_bin_path}" tfupdate
-    else
-      curl -L -C - "https://github.com/minamijoyo/tfupdate/releases/download/v${version}/tfupdate_${version}_linux_amd64.tar.gz"  | tar zxf - -C "${install_bin_path}" tfupdate
-    fi
+    curl -L -C - "https://github.com/minamijoyo/tfupdate/releases/download/v${version}/tfupdate_${version}_${plaform}_${arch}  .tar.gz"  | tar zxf - -C "${install_bin_path}" tfupdate
```
Comment (yuokada) [inline bin/install]:
```suggestion
    curl -L -C - "https://github.com/minamijoyo/tfupdate/releases/download/v${version}/tfupdate_${version}_${platform}_${arch}.tar.gz"  | tar zxf - -C "${install_bin_path}" tfupdate
```

**Comment by cgroschupp** [comment]
sorry for the typo.

**Comment by yuokada** [comment]
It works well on the machine with arm64 and linux. (I don't have any M1 mac machines. So, I can't test on it.)

[Testing plugins](https://asdf-vm.com/plugins/create.html#testing-plugins)
```
$ asdf plugin test tfupdate https://github.com/yuokada/asdf-tfupdate.git --asdf-plugin-gitref accaa8543916a9aa9219b2a3e215b7747a31d03f tfupdate --version
Updating tfupdate to accaa8543916a9aa9219b2a3e215b7747a31d03f
From https://github.com/yuokada/asdf-tfupdate
 * [new ref]         accaa8543916a9aa9219b2a3e215b7747a31d03f -> accaa8543916a9aa9219b2a3e215b7747a31d03f
warning: refname 'accaa8543916a9aa9219b2a3e215b7747a31d03f' is ambiguous.
Git normally never creates a ref that ends with 40 hex characters
because it will be ignored when you just specify 40-hex. These refs
may be created by mistake. For example,

  git checkout -b $br $(git rev-parse ...)

where "$br" is somehow empty and a 40-hex ref is created. Please
examine these refs and maybe delete them. Turn this message off by
running "git config advice.objectNameWarning false"
warning: refname 'accaa8543916a9aa9219b2a3e215b7747a31d03f' is ambiguous.
Git normally never creates a ref that ends with 40 hex characters
because it will be ignored when you just specify 40-hex. These refs
may be created by mistake. For example,

  git checkout -b $br $(git rev-parse ...)

where "$br" is somehow empty and a 40-hex ref is created. Please
examine these refs and maybe delete them. Turn this message off by
running "git config advice.objectNameWarning false"
Switched to branch 'accaa8543916a9aa9219b2a3e215b7747a31d03f'
tfupdate/bin/list-all is using GitHub API, just be sure you provide an API Authorization token
via your CI env GITHUB_API_TOKEN. This is the current rate_limit:

{
  "resources": {
    "core": {
      "limit": 60,
      "remaining": 42,
      "reset": 1641564182,
      "used": 18,
      "resource": "core"
    },
    "graphql": {
      "limit": 0,
      "remaining": 0,
      "reset": 1641564815,
      "used": 0,
      "resource": "graphql"
    },
    "integration_manifest": {
      "limit": 5000,
      "remaining": 5000,
      "reset": 1641564815,
      "used": 0,
      "resource": "integration_manifest"
    },
    "search": {
      "limit": 10,
      "remaining": 10,
      "reset": 1641561275,
      "used": 0,
      "resource": "search"
    }
  },
  "rate": {
    "limit": 60,
    "remaining": 42,
    "reset": 1641564182,
    "used": 18,
    "resource": "core"
  }
}

∗ Downloading and installing tfupdate...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   672  100   672    0     0  30545      0 --:--:-- --:--:-- --:--:-- 30545
100 3551k  100 3551k    0     0  24.0M      0 --:--:-- --:--:-- --:--:-- 24.0M
The installation was successful!
0.6.3
```

**Review by yuokada — APPROVED** [review]
Thanks for your contribution.

**Comment by cgroschupp** [comment]
```
uname -m
arm64

uname
Darwin

asdf plugin test tfupdate https://github.com/yuokada/asdf-tfupdate.git --asdf-plugin-gitref accaa8543916a9aa9219b2a3e215b7747a31d03f tfupdate --version
Updating tfupdate to accaa8543916a9aa9219b2a3e215b7747a31d03f
From ssh://github.com/yuokada/asdf-tfupdate
 * [new ref]         accaa8543916a9aa9219b2a3e215b7747a31d03f -> accaa8543916a9aa9219b2a3e215b7747a31d03f
warning: refname 'accaa8543916a9aa9219b2a3e215b7747a31d03f' is ambiguous.
Git normally never creates a ref that ends with 40 hex characters
because it will be ignored when you just specify 40-hex. These refs
may be created by mistake. For example,

  git switch -c $br $(git rev-parse ...)

where "$br" is somehow empty and a 40-hex ref is created. Please
examine these refs and maybe delete them. Turn this message off by
running "git config advice.objectNameWarning false"
warning: refname 'accaa8543916a9aa9219b2a3e215b7747a31d03f' is ambiguous.
Git normally never creates a ref that ends with 40 hex characters
because it will be ignored when you just specify 40-hex. These refs
may be created by mistake. For example,

  git switch -c $br $(git rev-parse ...)

where "$br" is somehow empty and a 40-hex ref is created. Please
examine these refs and maybe delete them. Turn this message off by
running "git config advice.objectNameWarning false"
Switched to branch 'accaa8543916a9aa9219b2a3e215b7747a31d03f'
tfupdate/bin/list-all is using GitHub API, just be sure you provide an API Authorization token
via your CI env GITHUB_API_TOKEN. This is the current rate_limit:

{
  "resources": {
    "core": {
      "limit": 60,
      "remaining": 54,
      "reset": 1641562284,
      "used": 6,
      "resource": "core"
    },
    "graphql": {
      "limit": 0,
      "remaining": 0,
      "reset": 1641565171,
      "used": 0,
      "resource": "graphql"
    },
    "integration_manifest": {
      "limit": 5000,
      "remaining": 5000,
      "reset": 1641565171,
      "used": 0,
      "resource": "integration_manifest"
    },
    "search": {
      "limit": 10,
      "remaining": 10,
      "reset": 1641561631,
      "used": 0,
      "resource": "search"
    }
  },
  "rate": {
    "limit": 60,
    "remaining": 54,
    "reset": 1641562284,
    "used": 6,
    "resource": "core"
  }
}

∗ Downloading and installing tfupdate...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   673  100   673    0     0   2654      0 --:--:-- --:--:-- --:--:--  2758
100 3990k  100 3990k    0     0  3267k      0  0:00:01  0:00:01 --:--:-- 6609k
The installation was successful!
0.6.3
```

**Pull request merged**